html/about.html

```diff
@@ -0,0 +1,168 @@
+            </li>
+            <li class="webpages">
+              <a href="about.html">About</a>
+            </li>
+            <li class="webpages">
+              <a href="#">Program</a>
+            </li>
+            <li class="webpages">
+              <a href="#">Sponsor</a>
+            </li>
+            <li class="webpages">
+              <a href="#">News</a>
+            </li>
+          </ul>
+
+          <div class="company-name">
+            <a href="index.html">CIC Campaign</a>
+          </div>
+        </nav>
+      </div>
+    </header>
+
+    <section class="first-section">
+      <div class="first-section-container">
+        <h2 class="open-intro-heading">"Hello! Connect, Learn & Grow"</h2>
+        <h1 class="open-heading">
+          CEOs International Community Global Summit 2021
+        </h1>
+        <div class="first-section-para">
+          <p>
+            The Global Summit brings together the community of experts,
+            academics, and activists who comprise the Creative Commons affiliate
+            network in a different country every two years. This year, we hope
+            to expand our invitation list including organizations and
+            invidivuals who want to work with us on shared projects that advance
+            the cause of the Commons, free culture and open knowledge. So if
+            you're active and engaged in the worlds of open content and
+            knowledge - free software advocates, Wikipedians. Open Knowledge,
+            galleries, libraries, museums, archives, governments and
+            foundations, lawyers, and activists - we hope you'll consider
+            joining us this year to build a stronger, more vibrant commons
+            together.
+          </p>
+        </div>
+        <div class="event-details">
+          <p>
+            Please contact us for any further questions about CEOs International
+            Community Global Summit 2021!
+          </p>
+          <a href=""
+            ><p class="email-address">cicglobalsummit2021@gmail.com</p></a
+          >
+        </div>
+      </div>
+    </section>
+
+    <section>
+      <div class="flex-content">
+        <h2>CEOs International Community Global Summit 2021 Logo</h2>
+        <hr />
+        <p class="mobile-para">
+          The logo of CIC Global Summit 2021 was decided through the logo
+          competition from 8 June to 7 July.
+        </p>
+        <p class="desktop-para">
+          The logo of CIC Global Summit 2021 was decided through the logo
+          competition from 8 June to 7 July. After online public voting and
+          Selection Committee's judging process, the logo from Naresh Agrawal
+          won the prize. After consultation with Mr. Agrawal we optimized the
+          prize-winning-logo for this year's summit as follows:
+        </p>
+      </div>
+      <div class="logo-container">
+        <img src="/assets/cic_logoReal1.png" alt="logo" />
+      </div>
```

Comment:
- [ ] The image is overflowing outside of the container in mobile view making a horizontal line in mobile view. Kindly fix this
![hj](https://user-images.githubusercontent.com/33067256/129426703-692efa12-41b0-4207-ad3d-b74fbf0b8975.png)